static/src/content/doc/single-node.md

```diff
@@ -0,0 +1,134 @@
+
+Regardless of your deployment environment, `bi` provides a consistent interface
+for:
+
+- **Managing installations**: Starting, stopping, and configuring environments.
+- **Database operations**: Creating connections, port forwarding, and retrieving
+  credentials.
+- **Network management**: Configuring VPN access to secure cluster resources.
+- **Debugging**: Collecting logs and diagnostic information.
+- **Cluster operations**: Interacting with your Kubernetes resources.
+
+`bi` combines multiple specialized tools into one streamlined interface that
+works the same way across all environments. As you progress through development
+to production, the same commands you've mastered locally will apply to your
+production environment.
+
+## Debugging Made Easy
+
+Connecting to databases is simple with built-in port forwarding:
+
+```bash
+bi postgres port-forward my-installation my-database-cluster -l 5432
+```
+
+For connection details:
+
+```bash
+bi postgres access-info my-installation my-database-cluster my-username --localhost
+```
+
+Troubleshooting is an essential part of development. Batteries Included provides
+multiple ways to debug your applications:
+
+- **Integrated Kubernetes views**: Access logs and resource usage directly from
+  the Control Panel.
+- **Access to pod logs**: View application output without complex kubectl
+  commands.
+- **Diagnostic tools**: Collect comprehensive debug information using the `rage`
+  command:
+
+```bash
+bi rage my-installation
+```
+
+## Enterprise-Ready Security
+
+Even in local development, Batteries Included maintains enterprise-grade
+security practices:
+
+- Each installation has a unique elliptic curve key for signing and encryption.
+- Communication between components uses public key encryption.
+- Credentials are handled securely with short-lived URLs that expire after
+  successful installation.
+
+## The Power of Consistency
+
+The real magic happens when your entire team uses the same tools and
+configurations. Code that runs locally will behave the same way in production
+because:
+
+- The same Kubernetes configurations are used across environments.
+- Service/battery configurations and versions remain consistent.
+- Networking configurations mirror production setups.
+- Security policies are enforced consistently.
+
+When it's time to deploy to production, you're not learning a new system—you're
+using the same familiar tools with different resource settings.
+
+## Beyond Local Development
+
+As your application grows, Batteries Included scales with you. You can retrieve
+WireGuard VPN configurations using:
+
+```bash
+bi vpn config my-production-installation
+```
+
+Batteries Included provides this consistent, easy-to-use toolset across all
+stages of development, making the transition from local to production as smooth
+as possible, and ensuring that what works locally will work in production just
+the same.
```

Comment:
This isn't something that users will be doing much at all. They don't need to know this to see that Batteries Included scales up. If we want to talk about the VPN then we should include that as a separate doc.